packages/shared/src/components/modals/streaks/StreakRecoverModal.tsx

```diff
@@ -0,0 +1,226 @@
+import React, { ReactElement, useId } from 'react';
+import { ModalSize } from '../common/types';
+import { ModalBody } from '../common/ModalBody';
+import { Modal } from '../common/Modal';
+import { LoggedUser } from '../../../lib/user';
+import {
+  Typography,
+  TypographyTag,
+  TypographyType,
+} from '../../typography/Typography';
+import {
+  Button,
+  ButtonProps,
+  ButtonSize,
+  ButtonVariant,
+} from '../../buttons/Button';
+import { anchorDefaultRel } from '../../../lib/strings';
+import { reputation as reputationHref } from '../../../lib/constants';
+import { ReputationIcon } from '../../icons';
+import { IconSize } from '../../Icon';
+import { useStreakRecover } from '../../../hooks/streaks/useStreakRecover';
+import { Checkbox } from '../../fields/Checkbox';
+import { ModalClose } from '../common/ModalClose';
+
+export interface StreakRecoverModalProps {
+  isOpen: boolean;
+  onRequestClose?: () => void;
+  user: LoggedUser;
+}
+
+const StreakRecoverCover = () => (
+  <div className="-mx-4 -mt-6" role="presentation" aria-hidden>
+    <svg viewBox="0 0 324 131" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Is this and svg from product or we can get a cloudinary link for it? If we're keeping it as svg let's add it in a file.

Reply:
Well for images we usually use links, this is more like an icon sth, so maybe we can just save it in its own file

Reply:
It should be an image I think actually it's cheaper to load from cloudinary then from vercel :)